test/username-decorator.test.js

```diff
@@ -0,0 +1,35 @@
+import chai from 'chai';
+import { decorateBareUsernameWithAt } from '../lib/helpers';
+chai.should();
+
+describe('Bare Slack Username Replacement', function() {
```

Comment:
Could use arrow functions instead of `function()` here (and below), since you aren't using `this` anywhere.